File: server/src/main/java/ua/com/alevel/service/finance/AccountingOfPaymentService.java
package ua.com.alevel.service.finance;

import org.springframework.stereotype.Service;
import ua.com.alevel.persistence.crudhelper.CrudRepositoryHelper;
import ua.com.alevel.persistence.entity.finance.AccountingOfPayment;
import ua.com.alevel.persistence.repository.finance.AccountingOfPaymentRepository;
import ua.com.alevel.service.AbstractService;

@Service
public class AccountingOfPaymentService extends AbstractService<AccountingOfPayment, AccountingOfPaymentRepository> {

    private final AccountingOfPaymentRepository accountingOfPaymentRepository;
    private final CrudRepositoryHelper<AccountingOfPayment, AccountingOfPaymentRepository> crudRepositoryHelper;

    public AccountingOfPaymentService(AccountingOfPaymentRepository accountingOfPaymentRepository,
                                      CrudRepositoryHelper<AccountingOfPayment, AccountingOfPaymentRepository> crudRepositoryHelper) {
        super(accountingOfPaymentRepository, crudRepositoryHelper);
        this.accountingOfPaymentRepository = accountingOfPaymentRepository;
        this.crudRepositoryHelper = crudRepositoryHelper;
    }
}

File: client-admin/src/app/shared/services/finance/accounting-of-payment.service.ts
import {Injectable} from '@angular/core';
import {AbstractService} from "../abstract.service";
import {HttpClient} from "@angular/common/http";
import {AccountingOfPayment} from "../../models/finance/accounting-of-payment.model";

@Injectable({
  providedIn: 'root'
})
export class AccountingOfPaymentService extends AbstractService<AccountingOfPayment> {

  override resourceUrl = '/api/accounting-of-payments';

  constructor(public override httpClient: HttpClient) {
    super(httpClient);
  }
}
